legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -67,6 +63,14 @@ def get_businesses(identifier: str):
 
         return jsonify(business=business_json)
 
+    auth_check_required = not current_app.config.get('BUSINESS_V2_PUBLIC')
+
+    # check authorization -- need to implement workaround for business search users before using this #22143
+    if auth_check_required and not authorized(identifier, jwt, action=['view']):
```

Comment:
Since it'll be hard to verify all the integration pieces that use this, can you add a warning log here that spits out the identifier, username, account id, and 'app-name' from the request headers if it exists?

Reply:
Also you can remove the part of the comment referencing the ticket and business search